template.yaml

```diff
@@ -247,6 +251,17 @@ Resources:
                   - secretsmanager:GetRandomPassword
                   - secretsmanager:ListSecrets
                 Resource: '*'
+        - PolicyName: ExternalUserPoolClients
```

Comment:
We encourage description for components we create in template.yaml

Reply:
Does not seem like its possible to add a Description to the Policy object. I did however add a Description to the Lambda after your feedback.